packages/next/client/index.js

```diff
@@ -294,11 +294,13 @@ function renderReactElement(reactEl, domEl) {
   }
 
   if (onPerfEntry && ST) {
-    if (!(PerformanceObserver in window)) {
-      window.addEventListener('load', () => {
-        performance.getEntriesByType('paint').forEach(onPerfEntry)
-      })
-    } else {
+    window.addEventListener('load', () => {
+      performance.getEntriesByType('paint').forEach(onPerfEntry)
+    })
+
```

Comment:
without an `else` this might relay twice.
once when through the observer and second on load.
thus the whole else thing is needed.

`window.addEventListener('load', () => {` is only to cover you when PerformanceObserver is not available

Reply:
Yep didn't think of that. Too enthralled with making it pretty :). Thanks